config/karma/test.karma.conf.js

```diff
@@ -11,8 +11,11 @@ const minimist = require('minimist');
 function getConfig(config) {
   require('./shared.karma.conf')(config);
 
-  const argv = minimist(process.argv.slice(2));
+  const argv = minimist(process.argv.slice(2), {
+    boolean: ['browser', 'enableDesktopNotifications']
```

Comment:
Why was `browser` added as a forced boolean?

Reply:
Because I'm an idiot. That's supposed to be `headless`. Fixed in my second commit.